index.html

```diff
@@ -64,5 +64,211 @@ <h3>LET'S CONNECT</h3>
+
+  <section class="headline about">
+    <div class="intro-phrase">
+          <h1>About<br>Myself</h1>
+        </div>
+        <div class="intro-paragraph">
+          <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <div class="connect-wrapper">
+          <h3>LET'S CONNECT</h3>
+          <div class="social-wrapper">
+            <ul class="social-list-wrapper">
+              <li>
+                <a href="https://twitter.com/knobaddy">
+                  <img class="social-icon-resize" src="assets/images/twitter-icon.png" alt="twitter-icon">
+                </a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/samuel-knower-amankwaa-frempong-356802256/">
+                  <img src="assets/images/Linkedin-icon.png" alt="LinkedIn-icon">
+                </a>
+              </li>
+              <li>
+                <a href="https://medium.com/@samankwaa.frempong">
+                  <img src="assets/images/mSocial-icon.png" alt="mSocial-icon">
+                </a>
+              </li>
+              <li>
+                <a href="https://github.com/frempongdev">
+                  <img src="assets/images/github-icon.png" alt="github-icon">
+                </a>
+              </li>
+              <li>
+                <a href="https://venture.angellist.com/v/samuel-amankwaa-frempong/i/portfolio">
+                  <img src="assets/images/handSocial-icon.png" alt="handSocial-icon">
+                </a>
+              </li>
+            </ul>
+          </div>
+      </div>
+      <div class="see-work resume-button">
+        <a href="#">Get my resume</a>
+      </div>
+      <div class="language-selection">
+        <h3>Languages</h3>
+        <a class="drop-down" href="#">
+          <img src="assets/mobile/down-button.png" alt="down-button">
+        </a>
+      </div>
+      <div class="technologies">
+        <div class="JS">
+          <img src="assets/mobile/Ellipse 1.png" alt="js-logo">
+          <h5>Javascript</h5>
+        </div>
+        <div class="HTML">
+          <img src="assets/mobile/Ellipse 2.png" alt="html-logo">
+          <h5>HTML</h5>
+        </div>
+        <div class="CSS">
+          <img src="assets/mobile/Ellipse 3.png" alt="css-logo">
+          <h5>CSS</h5>
+        </div>
+      </div>
+      <div class="language-selection">
+        <h3>Frameworks</h3>
+        <a class="drop-down" href="#">
+          <img src="assets/mobile/right-button.png" alt="down-button">
+        </a>
+      </div>
+      <div class="language-selection">
+        <h3>Skills</h3>
+        <a class="drop-down" href="#">
+          <img src="assets/mobile/right-button.png" alt="down-button">
+        </a>
+      </div>
```

Comment:
- [ ] Kindly I recommend to you use a `ul,` each of them in a `li` for a technology categories ("Languages", "Frameworks" and "Skills") and for The technology in each category(e.g., "JavasScript", "HTML" and "CSS") .